functions/pipeline/shared/onboarding/__init__.py

```diff
@@ -0,0 +1,53 @@
+import os
+import logging
+from azure.storage.blob import BlockBlobService
```

Comment:
this import is un-necessary, since you're not creating a new instance in this module - just being passed one

Reply:
Good point, forgot to delete